src/test/java/com.gigssandbox/UserMessengerTest.java

```diff
@@ -18,31 +19,26 @@
 
 
 @RunWith(MockitoJUnitRunner.class)
-public class ConsoleHelperTest {
+public class UserMessengerTest {
```

Comment:
add extra line after class declaration here. You have such extra line in most classes so lets follow a single code style rule everywhere